projects/lib-error-handler/tsconfig.lib.prod.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "./tsconfig.lib.json",
   "angularCompilerOptions": {
-    "enableIvy": false
+    "compilationMode": "partial"
```

Comment:
@Ashwin-op can you please confirm if this is correctly done?
@shinyenyi why have we switched to partial and removed ivy?

Reply:
this is recommended as angular 13 is 100% Ivy